webpack/assets/javascripts/react_app/components/ModelsTable/index.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import ModelsTable from './ModelsTable';
+import { schema } from './ModelsTableSchema';
+import reducer from './ModelsTableReducer';
+import * as actions from './ModelsTableActions';
+
+const mapStateToProps = ({ models }) => ({ ...models, schema });
```

Comment:
Why keeping the schema in the props?

Reply:
Again - you're correct. I don't think `schema` should be passed as props.
The `schema` has all the information on how the `Models Table` should be structured and thus I think it should be part of the component itself. I hope you agree with me too.